tests/data/samplesheets/samplesheet1.csv

```diff
@@ -1,4 +1,4 @@
-sample,mlst_alleles
-sample1,https://raw.githubusercontent.com/phac-nml/gasclustering/dev/tests/data/reports/sample1.mlst.json
-sample2,https://raw.githubusercontent.com/phac-nml/gasclustering/dev/tests/data/reports/sample2.mlst.json
-sample3,https://raw.githubusercontent.com/phac-nml/gasclustering/dev/tests/data/reports/sample3.mlst.json
+sample,mlst_alleles,metadata_1,metadata_2,metadata_3,metadata_4,metadata_5,metadata_6,metadata_7,metadata_8
+sample1,https://raw.githubusercontent.com/phac-nml/gasclustering/dev/tests/data/reports/sample1.mlst.json,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8
```

Comment:
Could you include two more test cases/samplesheets?

1. Where some of the metadata columns are missing values
2. Where all of the metadata columns are missing values

Reply:
Added in d1636fe15295e7b784916f0af959a0b360d83884

Reply:
Thanks so much.